### PR TITLE
USEID-113 Add Swagger for integration portal

### PR DESCRIPTION
**Pull request by pgurusinga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,3 +233,10 @@ See https://github.com/npryce/adr-tools regarding usage.
 
 Opt in to CI posting notifications for failing jobs to a particular Slack channel by setting a repository secret
 with the name `SLACK_WEBHOOK_URL`, containing a url for [Incoming Webhooks](https://api.slack.com/messaging/webhooks).
+
+## Swagger/OpenAPI
+
+How to access the page:
+
+1. [Production](https://useid.prod.ds4g.net/swagger/docs.html)
+2. [Staging](https://useid.dev.ds4g.net/swagger/docs.html)
```
Comment (mpanne) [inline README.md:242]: Add "3. [Local](http://localhost:8080/swagger/docs.html)"?

File: src/main/resources/static/eventSource.js [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
+const eventSource = new EventSource("/v1/sse");
 
 const getQRCode = document.getElementById("downloadQR");
 
 eventSource.addEventListener("ready", () => {
   getQRCode.src =
-    "/api/v1/qrcode/180?url=http%3A%2F%2Fwww.google.com" +
-    "&uuid=" +
-    event.data;
+    "/v1/qrcode/180?url=http%3A%2F%2Fwww.google.com" + "&uuid=" + event.data;
```
Comment (mpanne) [inline src/main/resources/static/eventSource.js:7]: Could remove the first +

**Review by mpanne — APPROVED** [review]
Cool, I like it. Just some small remarks.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+springdoc.api-docs.path=/v1
+springdoc.swagger-ui.path=/swagger/docs.html
+springdoc.swagger-ui.operationsSorter=method
```
[the remark this answers is not in the document]
Comment (pgurusinga) [reply src/main/resources/application.properties:3]: sure, I did not know that there's YAML version 🙈

[the remark this answers is not in the document]
Comment (mpanne) [reply src/main/resources/application.properties]: See my last commit. I migrated the config to YAML.

[the remark this answers is not in the document]
Comment (pgurusinga) [reply src/main/resources/application.properties]: @mpanne I also changed the property in the test directory

**Pull request merged**